packages/bot-strategy-runner/README.md

```diff
@@ -0,0 +1,96 @@
+# @uma/bot-strategy-runner
+
+This package contains a generalized bot runner enabling simple execution of multiple concurrent UMA bots on one machine.
+
+## Installing the package
+
+```bash
+yarn add @uma/bot-strategy-runner
+yarn build
+```
+
+Or, if you cloned from the mono repo simply run the following from the root of the mono repo:
+
+```bash
+yarn
+yarn qbuild
+```
+
+## Quick start
+
+This package requires a simple JSON config to parameterize the strategy runner. A simple config that enables a liquidator and disputer on 3 contracts would look like this:
+
+```json
+{
+  "globalAddressWhitelist": [
+    "0x0f4e2a456aAfc0068a0718E3107B88d2e8f2bfEF",
+    "0xd9af2d7E4cF86aAfBCf688a47Bd6b95Da9F7c838",
+    "0xd60139B287De1408f8388f5f57fC114Fb4B03328"
+  ],
+  "commonConfig": {
+    "priceFeedConfig": {
+      "cryptowatchApiKey": "YOUR-CRYPTO-WATCH-API-KEY",
+    },
+  "liquidatorSettings": { "enableBotType": true },
```

Comment:
I think the preview is broken because there's a typo in the json. I think this should fix it.

```suggestion
    }
  },
  "liquidatorSettings": { "enableBotType": true },
```